ZecHubInterview.md

```diff
@@ -0,0 +1,126 @@
+Another important point to make is that Zcash's implementation of Transparent Exchange Addresses (TEX) *does not* reduce user privacy or UX. We are in the process of implementing ZIP-320 which allows wallets to take shielded private funds and make an automatic two-step hop into Binance. 
+ 
+In the next 4 years regulation will deem onchain privacy a neccesity. There is already work being done to enable zk-identity an option on Zcash or provide asset issuers with encrypted visibility. You can be certain onchain transparency will be dealt with in the same way as unencrypted web traffic (HTTP vs HTTPS).
+
+## How does the inclusion of Zcash in the Bison Wallet (Decred decentralized exchange) align with the strategic goals of ZecHub and the Zcash community at large?
+
+Giving Zcash users a way to acquire and trade Zcash on a permmisionless & decentralised platform is absolutely necessary if the project is to retain its censorship resistance. This is how we protect our rights to financial sovereignty, free speech and privacy. Hyped for Bison Wallet and cannot wait to try it out + share instructional guides. 
+
+## Now talking about privacy! Privacy is a significant concern for many in the crypto space. How does ZecHub address these concerns, particularly in relation to the features of Zcash?
+
+ZecHub promotes the use of all privacy enhancing technologies, because Privacy is Normal. Everyone has the right to be able to choose what information is known about them. Zcash allows users the choice. You can choose when and with whom to send payments to publicly or privately. 
+In the modern day there is growing apathy towards large corporations harvesting user data through apps, websites, devices ..even your car! This data is used to sway elections and is often misused or made accessible to criminals! 
+ZecHub presents options for safely managing your data. Whether onchain using Zcash or online using Brave Browser, Tor or Graphene OS. 
+
+## Looking towards the future, what developments or advancements are most exciting at ZecHub?
+Over the next 6 months we will move to increase the size Bounties offered! As mentioned earlier we are looking to work with developers who are interested in Zcash to create proofs of concept demo apps. Engaging with other crypto projects is always quite fun, hopefully Decred, Digibyte, Litecoin and Firo can get together for another podcast soon ;)
+
+## Finally, what advice would you give to new users or investors interested in Zcash, Decred, or other cryptos in general, especially in the current market conditions?
+
+Unfortunately I am unable to provide financial or investment advice. However, I would always recommend for anyone interested in Decred or Zcash to try it out! The best research is done through experimentation - Zcash wallets are always fun to send encrypted messages with. Send me your shielded address and you may get an invite to the secret memo group.  
+
```

Comment:
Great article. I only have one nit pick and that is the paragraph formatting of this piece. Some times you add a line break between paragraphs and other time you don't . I would suggest adding the line breaks for consistence.

These formatting issues can be found at the end of lines: 75, 94, 95, 104, 105, 116, 117, 120

Reply:
Once you've completed these edits, I'm happy for you to upload to CT.